### PR TITLE
Section label wizardlet

### PR DESCRIPTION
**Pull request by zahnno**
Fixes issue when title of capability payload is a function

### CODE REVIEW

File: src/foam/u2/wizard/StepWizardletStepsView.js [hunk 1]
```diff
@@ -168,15 +167,22 @@ foam.CLASS({
         }))
     },
     function renderSectionLabel(elem, section, index, isCurrent) {
-      let title = section.navTitle;
-      if ( ! title || ! title.trim() ) title = "Part " + index;
-      return elem       
+      let title = section.title;
+      if ( ! title || ! foam.Function.isInstance(title) && ! title.trim() ) title = "Part " + index;
```
Comment (kgrgreer) [inline src/foam/u2/wizard/StepWizardletStepsView.js]: Could you not call .add() in this 'if' block to avoid creating a slot when not needed?

Comment (kgrgreer) [inline src/foam/u2/wizard/StepWizardletStepsView.js]: Should "Part" come from a Message?

Comment (zahnno) [reply src/foam/u2/wizard/StepWizardletStepsView.js]: Yup

Comment (zahnno) [reply src/foam/u2/wizard/StepWizardletStepsView.js]: A slot isn't created if title isn't a function. I've renamed the var to avoid confusion

**Pull request merged**